blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -32,6 +32,36 @@ const StyledLink = styled.a`
   color: ${(props) => props.primaryColor};
 `;
 
+// IMPROVEMENT: margin and margin bottom rem is reusable
+const FloatableImageContainer = styled.figure`
+  margin: 0 0 0 1rem;
```

Comment:
This is applying a margin left to all images in the article body and causing non floated images to not align as the currently do with the article text.

<img width="117" alt="Microsoft Edge-2021-07-19-11-06 55" src="https://user-images.githubusercontent.com/868127/126143877-9f253f6c-3db4-469d-857b-edaf8a123a54.png">

Reply:
thanks, this was a syntax mistake should be only bottom

Reply:
updated